data/mods/gen6xy/pokedex.ts

```diff
@@ -0,0 +1,42 @@
+export const Pokedex: {[k: string]: ModdedSpeciesData} = {
+	pikachu: {
+		inherit: true,
+		formeOrder: ["Pikachu"],
+	},
+	articuno: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	zapdos: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	moltres: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	skrelp: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	dragalge: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	tyrunt: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	tyrantrum: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	amaura: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
+	aurorus: {
+		inherit: true,
+		unreleasedHidden: true,
+	},
```

Comment:
Hidden ability gen 5 starters weren't released until ORAS came out, so they should probably be marked as unreleased in here